src/components/Journey.js

```diff
@@ -204,6 +204,7 @@ class Journey extends Component {
                 mapSrc: "/images/journeymap/journey-map_respondent-leave-to-appeal-refused.png",
                 userQuestion: this.respondentQuestion,
                 userState: "Notice of Leave to Appeal",
+                title: "Final Decision on Leave to Appeal",
```

Comment:
I'm thinking this prop should be renamed from 'title' to 'completeAppealPopupTitle'... longwinded but 'title' doesn't really make sense on the journeymap component until you get to creating the appeal complete popup

Reply:
sure, go!